airbyte-server/src/main/java/io/airbyte/server/handlers/WebBackendCheckUpdatesHandler.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.server.handlers;
+
+import io.airbyte.api.model.generated.WebBackendCheckUpdatesRead;
+import io.airbyte.config.StandardDestinationDefinition;
+import io.airbyte.config.StandardSourceDefinition;
+import io.airbyte.config.persistence.ConfigRepository;
+import io.airbyte.server.services.AirbyteGithubStore;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@AllArgsConstructor
+@Slf4j
+public class WebBackendCheckUpdatesHandler {
+
+  final ConfigRepository configRepository;
+  final AirbyteGithubStore githubStore;
+
+  public WebBackendCheckUpdatesRead checkUpdates() {
+
+    final int destinationDiffCount = getDestinationDiffCount();
+    final int sourceDiffCount = getSourceDiffCount();
+
+    return new WebBackendCheckUpdatesRead()
+        .destinationDefinitions(destinationDiffCount)
+        .sourceDefinitions(sourceDiffCount);
+  }
+
+  private int getDestinationDiffCount() {
+    final List<Entry<UUID, String>> currentActorDefToDockerImageTag;
+    final Map<UUID, String> newActorDefToDockerImageTag;
+
+    try {
+      currentActorDefToDockerImageTag = configRepository.listStandardDestinationDefinitions(false)
+          .stream()
+          .map(def -> Map.entry(def.getDestinationDefinitionId(), def.getDockerImageTag()))
+          .toList();
+    } catch (final IOException e) {
+      log.error("Failed to get current list of standard destination definitions", e);
+      return 0;
```

Comment:
Nit:  consider extracting to a constant to avoid re-use/magic number.

Reply:
done